src/layout/header/Header.tsx

```diff
@@ -15,10 +15,16 @@ const MainHeader = () => {
     <Header className={cn('main-header', { dark: isDarkTheme })}>
       <MenuOutlined onClick={() => setDrawerOpen(true)} className="hideOnDesktop" />
       <div style={{ flex: 1 }}>&nbsp;</div>
+      <GitHubLink />
       <button
+        className="theme-icon"
         onClick={toggleTheme}
         style={{ border: 'none', background: 'transparent', cursor: 'pointer' }}>
-        {isDarkTheme ? <BulbOutlined style={{ color: '#fff' }} /> : <BulbFilled />}
+        {isDarkTheme ? (
+          <BulbOutlined style={{ color: '#fff', fontSize: '1.5em' }} />
+        ) : (
+          <BulbFilled style={{ fontSize: '1.5em' }} />
+        )}
       </button>
```

Comment:
it could become a component

Reply:
I'll open an issue for this.  What do you think about using SVG's for the icons instead of importing from antd? 
@NoamGaash @shootermv

Reply:
Feel free to do so, gradually moving from antd to mui and other solutions is great.

Reply:
Also - at dark mode all the icons should be light, and at light they should be bark color
(This is something we already have at antd out of the box )